Purchases.xcodeproj/project.pbxproj

```diff
@@ -1654,6 +1651,24 @@
 			shellPath = /bin/sh;
 			shellScript = "scripts/swiftlint.sh\n";
 		};
+		F585B8BC269F74A300A54833 /* check TARGET_OS macro */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			name = "check TARGET_OS macro";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "#!/usr/bin/env bash\n\nif grep -l \"TARGET_OS\" $(find \"${PROJECT_DIR}/PurchasesCoreSwift\" -name \"*.swift\")\nthen\n  echo \"error: TARGET_OS macro has been found in some files\"\n  exit 1\nfi\n";
```

Comment:
We're likely going to rename `PurchasesCoreSwift` eventually, so maybe we can just search through the entire project folder (including the `Purchases`) folder for swift files. That way, if we change folder structure and forget about this, it will still catch things.